test/curator_test.rb

```diff
@@ -0,0 +1,162 @@
+    @curator.add_artist(@artist_1)
+    assert_equal [@artist_1], @curator.artists
+    @curator.add_artist(@artist_2)
+    assert_equal [@artist_1, @artist_2], @curator.artists
+  end
+
+  def test_method_find_artist_by_id
+    @curator.add_artist(@artist_1)
+    assert_equal @artist_1, @curator.find_artist_by_id("1")
+    @curator.add_artist(@artist_2)
+    assert_equal @artist_2, @curator.find_artist_by_id("2")
+  end
+
+  def test_method_find_photograph_by_id
+    @curator.add_photograph(@photo_1)
+    assert_equal @photo_1, @curator.find_photograph_by_id("1")
+    @curator.add_photograph(@photo_2)
+    assert_equal @photo_2, @curator.find_photograph_by_id("2")
+  end
+
+  def test_method_find_photographs_by_artist
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+    assert_equal [@photo_3, @photo_4], @curator.find_photographs_by_artist(@artist_3)
+  end
+
+  def test_method_artists_with_multiple_photographs
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+    assert_equal [@artist_3], @curator.artists_with_multiple_photographs
+  end
+
+  def test_method_photographs_taken_by_artists_from
+    @curator.add_artist(@artist_1)
+    @curator.add_artist(@artist_2)
+    @curator.add_artist(@artist_3)
+    @curator.add_photograph(@photo_1)
+    @curator.add_photograph(@photo_2)
+    @curator.add_photograph(@photo_3)
+    @curator.add_photograph(@photo_4)
+    assert_equal [@photo_2, @photo_3, @photo_4], @curator.photographs_taken_by_artists_from("United States")
+    assert_equal [], @curator.photographs_taken_by_artists_from("Argentina")
+  end
+
+  def test_method_load_photographs
+    @curator.load_photographs('./data/photographs.csv')
+    assert_equal 4, @curator.photographs.length
+    assert_equal "1", @curator.photographs[0].id
+    assert_equal "Rue Mouffetard, Paris (Boy with Bottles)", @curator.photographs[0].name
+    assert_equal "1", @curator.photographs[0].artist_id
+    assert_equal "1954", @curator.photographs[0].year
+  end
+
+  def test_load_artists
+    @curator.load_artists('./data/artists.csv')
+    assert_equal 6, @curator.artists.length
+    assert_equal "1", @curator.artists[0].id
+    assert_equal "Henri Cartier-Bresson", @curator.artists[0].name
+    assert_equal "1908", @curator.artists[0].born
+    assert_equal "2004", @curator.artists[0].died
+    assert_equal "France", @curator.artists[0].country
+  end
+
+  def photographs_taken_between
+    @curator.load_photographs('./data/photographs.csv')
+    assert_equal 2, @curator.photographs_taken_between(1950..1965).length
```

Comment:
would have been better, like you did above, to look at the photos that came back from this file and got precisely which photos you expected to see here